internal/tfimport/tfimport.go

```diff
@@ -577,6 +582,7 @@ func planAndImport(rn, importRn runner.Runner, runArgs *RunArgs) (retry bool, er
 	var errs []string
 	var skipped []string
 	var importCmds []string
+	var notImportable []string
```

Comment:
`notImportableMsgs`

Reply:
This PR is built on the other one, I'll fix that one first and rebase this one.